src/main/java/org/commcare/formplayer/application/MenuController.java

```diff
@@ -63,6 +64,19 @@ public class MenuController extends AbstractBaseController {
 
     private final Log log = LogFactory.getLog(MenuController.class);
 
+    @RequestMapping(value = Constants.URL_INTERVAL_SYNC_DB, method = RequestMethod.POST)
+    @UserLock
+    @UserRestore
+    @AppInstall
+    public SyncDbResponseBean scheduleSync(@RequestBody SessionNavigationBean sessionNavigationBean,
+            @CookieValue(Constants.POSTGRES_DJANGO_SESSION_ID) String authToken,
+            HttpServletRequest request) throws Exception {
+        if (restoreFactory.isRestoreXmlExpired()) {
+            restoreFactory.performTimedSync();
+        }
+        return new SyncDbResponseBean();
```

Comment:
I'm not sure what's happening on the front end but seems like it could be useful to indicate whether a sync actually happened or not in the response.

Reply:
makes sense, [2458e1e](https://github.com/dimagi/formplayer/pull/1544/commits/2458e1e6a4477b673133291cdd0e38321612ba2d)